lib/models/BaseNode.ts

```diff
@@ -6,6 +6,7 @@ import {BlueshellState, NodeStorage, ResultCode} from '../models';
 export interface BaseNode<S extends BlueshellState, E> {
 	name: string;
 	parent: string;
+	id: string;
```

Comment:
is this readonly? or should there be a setter in Base.ts

Reply:
Should be read-only.  Updated (next push)